helpers/errorCodes.js

```diff
@@ -0,0 +1,126 @@
+module.exports = {
+
+  // E00 - General errors
+  E0000: {
+    code: 'E0000',
+    message: 'Unknown error'
+  },
+  E0001: {
+    code: 'E0001',
+    message: 'Authentication token is invalid or expired.'
+  },
+  E0002: {
+    code: 'E0002',
+    message: 'Users role does not have the necessary permissions for this action'
+  },
+
+  // E01 - Login errors
+  E0101: {
```

Comment:
We've made some changes in the error codes, which we also need to add to Resources and Educado-mobile. E.g. E0101 and E0105

(It might also be nice to have these next to each other while it's still feasible to be making changes. Though I know that shouldn't really be a priority right now)

Reply:
Request fixed